instantlaunches/db.go

```diff
@@ -444,20 +450,19 @@ const listPublicQLsQuery = `
 	SELECT ql.id,
 		u.username as creator,
 		ql.app_id,
+		ql.app_version_id,
 		ql.name,
 		ql.description,
 		ql.is_public,
 		s.submission
 	FROM quick_launches ql
-	JOIN app_listing a on ql.app_id = a.id
 	JOIN users u on ql.creator = u.id
 	JOIN submissions s on ql.submission_id = s.id
-	WHERE u.username = $1 OR ( ql.is_public = true AND a.is_public = true)
+	WHERE u.username = $1 OR ql.is_public = true
```

Comment:
The `app_listing.is_public` flag is obsolete and will be dropped by cyverse-de/de-database#10.